managed-os/amilookup.js

```diff
@@ -10,7 +10,14 @@ var osNameToPattern = {
     "Windows Server 2012 64-bit": "Windows_Server-2012-RTM-English-64Bit-Base-*",
     "Windows Server 2012 R2 64-bit": "Windows_Server-2012-R2_RTM-English-64Bit-Base-*",
     "Windows Server 2016 Base": "Windows_Server-2016-English-Full-Base-*",
-    "Red Hat Enterprise Linux 7.2": "RHEL-7.2_HVM_GA*"
+    "Red Hat Enterprise Linux 7.2": "RHEL-7.2_HVM_GA*",
+    "CIS Windows Server 2008 SP2 32-bit": "CIS Microsoft Windows Server 2008 Benchmark*",
+    "CIS Windows Server 2008 SP2 64-bit": "CIS Microsoft Windows Server 2008 Benchmark*",

```

Comment:
is it same image used for 32-bit and 64-bit?

Reply:
@RajkumarShivage I checked but there is no 32 bit CIS hardened image. The Architecture of windows OS is of "x86_64" type. So should i remove 32 bit windows 2008 key from the map.

Reply:
@RajkumarShivage  addressed by removing the 32 bit windows 2008 key from the map for CIS hardened image